Hydro/src/Hydro/Renderer/Renderer.cpp

```diff
@@ -38,7 +39,7 @@ namespace Hydro
 		s_Instance->m_vulkanPresentation->CreateSwapChain(window, true);
 
 		VulkanAllocator::Init(s_Instance->m_rendererContext->GetVulkanDevice());
-		// Renderer2D::Init();
-		Renderer3D::Init();
+		Renderer2D::Init();
+		// Renderer3D::Init();
```

Comment:
should also be possbile to be inited for this we need to either create seperate shaders or make a shader library that can reference certain shaders.